### DISCUSSION TITLE
Onboarding PR not created due to rate-limit-exceeded on Github

### DISCUSSION THREAD

**Discussion by cfzlp** [renovatebot/renovate#20339] [Request Help] [answered]
### I'm using the Mend Renovate hosted app.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

Hi!

We installed Renovate on "All repositories" in our GitHub org. Renovate did not create onboarding PRs in a few repositories. For each of these repositories, the dashboard shows only on job with the result "rate-limit-exceeded".
It created the `renovate/configure` branch. Then, it is about to create the onboarding PR and it hits the GitHub API rate limit.

Is there a way to trigger Renovate to try to onboard this repository again? Will Renovate retry?

Note: uninstalling/reinstalling Renovate on "All repositories" is an "expensive" operation because it removes Renovate from some branch protection rules that we have in place. We would like to avoid that, if possible.

Thanks for any help.

Carlos

### Relevant debug logs.

<details><summary>Logs</summary>

```
DEBUG: getPrList()
DEBUG: Creating onboarding PR
```
...
```
WARN: Rate limit exceeded - aborting
DEBUG: Repository result: rate-limit-exceeded, status: onboarding, enabled: true, onboarded: false
```

</details>

**Comment by viceice** [comment]
i think you probably just need to wait some time to get it happen.

**Comment by rarkins** [comment, marked as answer]
Please tell me a couple of the repo names or Job IDs. I would expect it to retry after a few hours, but will double check

**Comment by cfzlp** [comment]
Two such repos are
- cognitedata/profiler-api
- cognitedata/extractor-download-api

Yesterday I could see one job for each of them, with result "rate-limit-exceeded", as I described in my first post, but now there aren't any jobs listed for either of the repos.

Thanks for looking into it.

**Reply by rarkins** [reply]
I have triggered these all to run again although will try to stagger them out through multiple hours of today so they don't exceed rate limits

**Reply by cfzlp** [reply]
Thank you.
